app/common/directives/ush-logo-react/ush-logo.jsx

```diff
@@ -1,14 +1,16 @@
-import React, { Component } from 'react';
-import { render } from 'react-dom';
+import React from 'react';
```

Comment:
This is placeholder code to test the new linter, yeah?

Reply:
No, these were the changes that the linter made me make to fit eslint/airbnb's react styles. Should have made that clea